Content.Shared/GameObjects/Components/Movement/MovementIgnoreGravityComponent.cs

```diff
@@ -14,12 +15,15 @@ public sealed class MovementIgnoreGravityComponent : Component
 
     public static class GravityExtensions
     {
+        public static Action<IEntity,bool> ?OnWeightlessChanged;
```

Comment:
nitpick, but looks good imo.

dunno how i feel about the global event, that might be costy real quick. might be better to add a local event somewhere on the entity?
```suggestion
        public static Action<IEntity,bool>? OnWeightlessChanged;
```

Reply:
> somewhere on the entity?

To the IEntity interface? if so, i did consider it for a minute.

Reply:
no never ever do something like that on an entity.

if anything the options would be
a) find a component that relates to weighlessness and add it on there
b) create a new component to hold the event

Reply:
hold on i got an idea

Reply:
also just realized you are using `Action<IEntity,bool>?`. this means you can only subscribe one (1) method to this "event"